/**:
  ros__parameters:
    global_slope_max_angle_deg: 10.0
    local_slope_max_angle_deg: 13.0 # recommended 30.0 for non elevation_grid_mode
    split_points_distance_tolerance: 0.2
    use_virtual_ground_point: True
    split_height_distance: 0.2
    non_ground_height_threshold: 0.20
    grid_size_m: 0.5
    grid_mode_switch_radius: 20.0
    gnd_grid_buffer_size: 4
    detection_range_z_max: 2.5
    elevation_grid_mode: true
    low_priority_region_x: -20.0
    center_pcl_shift: 0.0
    radial_divider_angle_deg: 1.0
    use_recheck_ground_cluster: true
    use_lowest_point: true

    # debug parameters
    publish_processing_time_detail: false
